src/components/Personal-Zone/Searcher/AddResumeDialog/ProgressBar/ProgressBar.tsx

```diff
@@ -59,34 +60,68 @@ const ProgressBar: React.FC<ProgressBarProps> = (props): JSX.Element => {
                     setPreviousJobs={setPreviousJobs}
                     jobName={jobName}
                     setJobName={setJobName} />;
-            case 2:
+            case ResumeStep.NEXT_JOB:
                 return <NextJob
                     roles={allSelectOptions?.roleOptions || []}
                     nextRoles={nextRoles}
                     setNextRoles={setNextRoles}
                     aboutMe={aboutMe}
-                    setAboutMe={setAboutMe} />;
+                    setAboutMe={setAboutMe}/>;
             default:
-                return 'Unknown step';
+                return <></>;
         }
     }
 
-    const handleNext = () => {
-        setActiveStep((prevActiveStep) => prevActiveStep + 1);
+    const handleNext = (): void => {
+        if (validateStep()) {
+            if (activeStep + 1 < steps.length) {
+                setActiveStep((prevActiveStep) => prevActiveStep + 1);
+            } else {
+                submitResume();
+            }
+        }
     };
 
-    const handleBack = () => {
+    const handleBack = (): void => {
         setActiveStep((prevActiveStep) => prevActiveStep - 1);
     };
 
+    const validateStep = (): boolean => {
+        switch (activeStep) {
+            case ResumeStep.PERSONAL_DETAILS: {
+                const alteredSteps: StepInfo[] = steps.slice();
+                alteredSteps[ResumeStep.PERSONAL_DETAILS].errors = {};
+
+                if (!(/^\d{4,10}$/.test(phoneNumber))) {
```

Comment:
why not 
```suggestion
                if (!(/^\d{10}$/.test(phoneNumber))) {
```
exactly 10, maybe we need to ask Yuval

Reply:
Either way, `export to const MAX_PHONE_LEN`

Reply:
done